lib/hashes.md

```diff
@@ -5,40 +5,42 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A hash is the python form of a dictionary.  They are similar to arrays, but whereas an array uses an integer to index and organize the objects located within it, whereas a hash is more variable.  It can use any value to help organize
+the list.  Hashes can help group different items/objects in the list into associations.  Like what I know from python, the key is the sorting, association-making variables that help group, label, and identify the values assigned to them.
```

Comment:
> It can use any value to help organize the list.

I want to urge you to be precise in your language here. Ruby Hashes (like Python Dictionaries) contain key-value pairs and are indexed on their _keys_.  Keys can be any data type.